src/System.Collections.NonGeneric/tests/CaseInsentiveComparerTests.cs

```diff
@@ -130,41 +131,40 @@ public static void Ctor_CultureInfo_NullCulture_ThrowsArgumentNullException()
         [InlineData(null, null, 0)]
         public static void DefaultInvariant_Compare(object a, object b, int expected)
         {
-            var cultureNames = new string[]
+            RemoteInvoke(() =>
```

Comment:
I'm thinking not. I originally added the RemoteInvoke with the intention of leaving them as DefaultThread but later changed them since it wouldn't make a difference either way.  The Process dependency also breaks the uap build. I'm removing them.